sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Removed lambda parameter from "if_architecture_is" according to [`sycl_ext_oneapi_device_architecture`](doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc). [71f745bcda7d]
+- Throw for invalid reqd_work_group_size. [084c02797084]
+- Adjusted static restriction checks for block_2d APIs. [7902d6b35348]
+- Disallowed local accessor misuse according to SYCL 2020. [d7a5ec048368]
+- Implemented passing address space information to SPIR-V Joint Matrix Load/Store intrinsics. [854ab7eab5e4]
+- Enabled immediate command lists by default for the Level Zero backend. [9e5b2ed1ddc3]
+- Removed extra map lookup for eliminated kernel arguments during enqueue of cached kernels. [7a1fa6c38e03]
+- Changed the default to `SYCL_PI_LEVEL_ZERO_USM_RESIDENT=2`. [85435666d477]
+- Added heuristics to reduce overhead from immediate command-list cleanup in the Level Zero backend. [ce69d6906e30]
+- Renamed `cluster_group` to `fixed_size_group`. [62c091a0af17]
+- Added error for invalid uniform arguments in InvokeSIMD. [487c1f8cb040]
+- Reduced overhead from queue profiling by using `steady_clock::now()` to retrieve host time as `piGetDeviceAndHostTimer` has large overhead. [070598e433a7]
+- Implemented graceful handling of the case that Level Zero runtime was already unloaded when we do cleanup. [928a919f15d9]
+- Improved error messages for InvokeSIMD. [0c74bbb20e65]
+- Updated `native_specialization_constant()` for the case when no specialization constants are present. [9eb1184686fb]
+- Optimized re-use of command lists in the Level Zero backend. [24ae0153e8e8]
+
+### Documentation
+- Updated restrictions in [`sycl_ext_oneapi_device_architecture`](doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc) extension. [a69a54f38f2f]
+- Added `invoke_simd` SLM example and `simd_view` example to [`README``](doc/extensions/supported/sycl_ext_intel_esimd/examples/README.md). [c99668ee5985][f1986b40e04b]
+- Clarified ways to set sub-group sizes in [`sycl_ext_oneapi_named_sub_group_sizes``](sycl/doc/extensions/proposed/sycl_ext_oneapi_named_sub_group_sizes.asciidoc) [3285e0fe5590]
+- Added invoke_simd example. [e860b140d506]
+- Added README.md for the Unified Runtime directory. [453ffdcf41dd]
+- Updated the versioning section in [`ABIPolicyGuide`](doc/developer/ABIPolicyGuide.md). [dfe6e4e542ab]
+- Added `invoke_simd` section to [`README`](sycl/doc/extensions/experimental/sycl_ext_intel_esimd/README.md) and updated [sycl_ext_intel_esimd](doc/extensions/experimental/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md) [52567e8cd428]
+- Updated [`sycl_ext_oneapi_device_architecture`](doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc) and [`sycl_ext_oneapi_device_if`](doc/extensions/proposed/sycl_ext_oneapi_device_if.asciidoc) to allow captures in `if_device_has`, `if_architecture_is`. [acfc905a994a]
+- Updated [SPV_INTEL_joint_matrix](doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc) extension. [adbbf0f39c25]
```

Comment:
```suggestion
- Updated [`SPV_INTEL_joint_matrix`](doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc) extension. [adbbf0f39c25]
```

SPIR-V extensions are not really user-visible. Not sure if it worth mentioning this. Tagging @MrSidims